public/js/auth.js

```diff
@@ -0,0 +1,7 @@
+function sign_up_user() {
+    window.location.href = "/register";
+}
+
+function sign_in_user() {
```

Comment:
Identifier 'sign_in_user' is not in camel case.<br>'sign_in_user' is defined but never used.